WalletWasabi.Tests/UnitTests/WabiSabi/Backend/UtxoPrisonTests.cs

```diff
@@ -95,4 +95,21 @@ public void PrisonOperations()
 		Assert.Equal(id2, inmate!.LastDisruptedRoundId);
 		Assert.True(p.TryRelease(utxo, out _));
 	}
+
+	[Fact]
+	public void CanReleaseAfterLongBan()
+	{
+		var p = new Prison();
+		var id1 = BitcoinFactory.CreateUint256();
+		var utxo = BitcoinFactory.CreateOutPoint();
+		var past = DateTimeOffset.UtcNow - TimeSpan.FromDays(40);
+
```

Comment:
Here, you should do imo:

```suggestion
Assert.Empty(p.GetInmates());
```